dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/utils/DependentExecute.java

```diff
@@ -337,18 +337,27 @@ private ProcessInstance findLastProcessInterval(Long definitionCode, Long taskCo
     }
 
     /**
-     * get dependent result by task/process instance state
+     * get dependent result by task/process instance
      *
-     * @param state state
+     * @param processInstance process instance
+     * @param taskInstance task instance
      * @return DependResult
      */
-    private DependResult getDependResultByState(TaskExecutionStatus state) {
+    private DependResult getDependResultOfTask(ProcessInstance processInstance, TaskInstance taskInstance) {
 
+        TaskExecutionStatus state = taskInstance.getState();
         if (!state.isFinished()) {
             return DependResult.WAITING;
         } else if (state.isSuccess()) {
             return DependResult.SUCCESS;
         } else {
+            log.info("Task code: {}, task name: {}, retryTimes: {}, maxRetryTimes: {}",
+                    taskInstance.getTaskCode(), taskInstance.getName(), taskInstance.getRetryTimes(),
+                    taskInstance.getMaxRetryTimes());
+            if (processInstance.getState().isRunning()
+                    && taskInstance.getRetryTimes() < taskInstance.getMaxRetryTimes()) {
+                return DependResult.WAITING;
+            }
```

Comment:
```suggestion
            if (processInstance.getState().isRunning()
                    && taskInstance.getRetryTimes() < taskInstance.getMaxRetryTimes()) {
                log.info("taskDefinitionCode: {}, taskDefinitionName: {}, retryTimes: {}, maxRetryTimes: {}",
                    taskInstance.getTaskCode(), taskInstance.getName(), taskInstance.getRetryTimes(),
                    taskInstance.getMaxRetryTimes());
                return DependResult.WAITING;
            }
```

Reply:
Okay, this can prevent prolonged checks from printing a large amount of duplicate logs

Reply:
Yes.